docs/_articles/en/einstein/einstein-inline.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Inline Automatic Completions
+lang: en
+---
+
+## Overview
+
+Use AI-based autocomplete to accept suggestions for repetitive, boilerplate code right in your editor.
+
+## Set Up Inline Autocomplete
+
+To use inline autocomplete, first activate the feature:
+
+1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
+2. Under **Einstein for Developers**, select **Enable Einstein Auto Completions**.
+
+## Receive your First Inline Completions
+
+Einstein for Developers autogenerates inline code completions suggestions. To start receiving inline completions:
+
+1. In Visual Studio Code, create a new Apex class (\*.cls) file, or open an existing one.
+
+2. In the Apex file, place your cursor inside a method. Einstein for Developers automatically suggests an entire method body in grayed text, as shown below. The exact suggestion may vary.
```

Comment:
In step two it says "inside a method" but one option in step 1 was to create a _new_ Apex class.

I don't think we necessarily want to highlight the new/empty file scenario because Inline doesn't have any context at that point and is unlikely to give a useful suggestion.

I think we should show users how to use this feature by highlighting specific use cases that we know are likely to provide useful suggesions, like maybe: 

> in an Apex class, start writing a new method - see E4D anticipate what you need and suggest some code for you.

or

> when writing apex code, you are often doing things like assigning variable values to an Sobject.  
> ie Account.name = abc, Account.type=Business, etc.
> pause the cursor after "Business,etc." and watch E4D complete your code for you and fill out the remaining fields.